api/src/wfl/module/aou.clj

```diff
@@ -240,34 +230,18 @@
                         set)]
       (set (remove existing samples)))))
 
-(comment
-  (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
-                            (let [samples [{:analysis_version_number     1,
-                                            :chip_well_barcode           "7991775143_R01C01",
-                                            :green_idat_cloud_path       "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Grn.idat",
-                                            :params_file                 "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/inputs/7991775143_R01C01/params.txt",
-                                            :red_idat_cloud_path         "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Red.idat"
-                                            :reported_gender             "Female"
-                                            :sample_alias                "NA12878"
-                                            :sample_lsid                 "broadinstitute.org:bsp.dev.sample:NOTREAL.NA12878"
-                                            :bead_pool_manifest_file     "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm"
-                                            :cluster_file                "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt"
-                                            :zcall_thresholds_file       "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/IBDPRISM_EX.egt.thresholds.txt"
-                                            :gender_cluster_file         "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt"
-                                            :extended_chip_manifest_file "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv"}]]
-                              (remove-existing-samples! tx samples "allofusarrays_000000001"))))
-
 (defn append-to-workload!
   "Use transaction TX to append the samples to a WORKLOAD identified by uuid.
    The workload needs to be marked as 'started' in order to be append-able, and
    any samples being added to the workload table will be submitted right away."
-  [tx {:keys [notifications uuid environment] :as body}]
+  [tx {:keys [notifications uuid environment]}]
```

Comment:
Where is the WORKLOAD?

Reply:
Suggest changing `body` to `_workload` to parallel other `…-workload!` methods.

Reply:
will do 👍